docusaurus/docs-internals/editor-controls/multivalue-controls/background-color.md

```diff
@@ -0,0 +1,208 @@
+| `states?`          | `Array<State>`                           | [`"normal"`] | Allows the control to work in different states. <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination) |
+| `config?.opacity`  | `boolean`                                |    `true`    | Controls whether the opacity of the background color can be changed                                                                                                                                                                                                                                                            |
+| `config?.withNone` | `boolean`                                |    `true`    | Determines whether the `None` option is included in the dropdown menu for background type                                                                                                                                                                                                                                      |
+
+
+### Return value
+
+Returns an object with the following values:
+
+`active` - a string indicating the currently selected handle when changing the start or end color of the gradient. It can be either `"start"` or `"end"`; <br/>
+`degree` - a number representing the angle of the gradient in degrees; <br/>
+`end` - a number indicating the position of the handle for the end color of the gradient on the gradient colors slider, where `100` represents the end of the slider and `0` represents the start; <br/>
+`gradientType` - a string that specifies the type of gradient. It can be either `"linear"` for a linear gradient or `"radial"` for a radial gradient; <br/>
+`hex` - a string representing the hex code of the selected color; <br/>
+`opacity` - a number representing the opacity level of the selected color; <br/>
+`palette` - a predefined palette from global styles; <br/>
+`start` - a number indicating the position of the handle for the start color of the gradient on the gradient colors slider, where `100` represents the end of the slider and `0` represents the start; <br/>
+`type` - a string that specifies the type of background color. It can be either `"solid"` for a solid color or `"gradient"` for a gradient color;
+
+```js
+{
+  active: "start" | "end";
+  degree: number;
+  end: number;
+  gradientType: "linear" | "radial";
+  hex: string;
+  opacity: 1;
+  palette: string;
+  start: number;
+  type: "solid" | "gradient";
```

Comment:
`gradientHex ` is missing 

to check all keys that return look at `toElementModel` here: `editor/visual/editor/js/utils/options/BackgroundColor/converters.ts`

Reply:
added `gradientHex`, `gradientPalette, `gradientOpacity`